pointercrate-user/src/video.rs

```diff
@@ -23,6 +23,16 @@ pub fn validate_channel(url: &str) -> Result<String> {
                 if let Some(path_segments) = url.path_segments() {
                     match &path_segments.collect::<Vec<_>>()[..] {
                         ["channel", _] | ["user", _] | ["c", _] => Ok(url.to_string()),
+                        [handle] => {
+                            if handle.starts_with("@") {
+                                Ok(url.to_string())
+                            } else {
+                                Err(CoreError::InvalidUrlFormat {
+                                    expected: YOUTUBE_CHANNEL_FORMAT,
+                                }
+                                .into())
+                            }
+                        }
```

Comment:
You can use a [match guard](https://doc.rust-lang.org/rust-by-example/flow_control/match/guard.html) here to avoid repeating the `return Err(...)` code

```suggestion
                        [handle] if handle.starts_with('@') => {
                            Ok(url.to_string())
                        }
```